tests/core/test_multi_cluster_app.py

```diff
@@ -281,6 +281,50 @@ def test_admin_access(admin_mc, admin_pc, user_factory, remove_resource):
     wait_for_roles_to_be_updated(admin_mc, updated_mcapp, ["cluster-owner"])
 
 
+def test_add_projects(admin_mc, admin_pc, admin_cc, remove_resource):
+    client = admin_mc.client
+    mcapp_name = random_str()
+    temp_ver = "cattle-global-data:library-wordpress-2.1.10"
+    targets = [{"projectId": admin_pc.project.id}]
+    mcapp1 = client.\
+        create_multi_cluster_app(name=mcapp_name,
+                                 templateVersionId=temp_ver,
+                                 targets=targets,
+                                 roles=["project-member"])
+    wait_for_app(admin_pc, mcapp_name, 60)
+    remove_resource(mcapp1)
```

Comment:
`remove_resource` should be called immediately after the object is created so if the `wait_for_app` fails your object is still cleaned up.